protocol/messenger_local_backup.go

```diff
@@ -258,6 +276,15 @@ func (m *Messenger) ExportBackup() ([]byte, error) {
 	for _, d := range chatsToBackup {
 		backup.Chats = append(backup.Chats, d.Chats...)
 	}
+
+	backupMessages, err := m.backupMessages()
+	if err != nil {
+		return nil, err
+	}
+	if backupMessages != nil {
+		backup.Messages = backupMessages
+	}
```

Comment:
Setting it to `nil` doesn't sound like an issue 🤔 
```suggestion
backup.Messages = backupMessages
```